scripts/galaxykickstart_from_workflow.py

```diff
@@ -0,0 +1,96 @@
+import os
+import sys
+import textwrap
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
+from shutil import copyfile
+from generate_tool_list_from_ga_workflow_files import *
+
+
+
+def _parse_cli_options():
+    """
+    Parse command line options, returning `parse_args` from `ArgumentParser`.
+    """
+    parser = ArgumentParser(formatter_class=RawDescriptionHelpFormatter,
+                            usage="python %(prog)s <options>",
+                            epilog="example:\n"
+                                   "python %(prog)s -w workflow1 workflow2 -l my_panel_label\n"
+                                   "Christophe Antoniewski <drosofff@gmail.com>\n"
+                                   "https://github.com/ARTbio/ansible-artimed/tree/master/scritps/galaxykickstart_from_workflow.py")
+    parser.add_argument('-w', '--workflow',
+                        dest="workflow_files",
+                        required=True,
+                        nargs='+',
+                        help='A comma-separated of galaxy workflow description files in json format', )
+    parser.add_argument('-l', '--panel_label',
+                        dest='panel_label',
+                        default='Tools from workflows',
+                        help='The name of the panel where the tools will show up in Galaxy.'
+                             'If not specified: "Tools from workflows"')
+    return parser.parse_args()
+
+def makedir (path):
+    try:
+        os.makedirs(path)
+    except OSError:
+        if not os.path.isdir(path):
+            raise
+
+def make_inventory (path="../inventory_files/"):
+    file_path = path + "gks_workflows"
+    try:
+        f = os.fdopen(os.open(file_path, os.O_CREAT | os.O_WRONLY | os.O_EXCL), 'w')
+    except OSError:
+        sys.exit("gks_workflows inventory file already exists in ../inventory_files")
+    print >> f, textwrap.dedent('''\
```

Comment:
I think this screams for using the string.Template module.
Have a look at https://github.com/galaxyproject/ansible-galaxy-extras/blob/master/templates/configure_slurm.py.j2.
The idea is that you define a template (SLURM_CONFIG_TEMPLATE, in that example), and you fill the template with `config_contents = Template(SLURM_CONFIG_TEMPLATE).substitute(template_params)`